tests/dflowfm/test_polyfile.py

```diff
@@ -268,7 +268,7 @@ def test_convert_to_comment(self, input: str, expected_value: str):
             ("*", False),
             ("*         ", False),
             ("", False),
-            ("Not a comment but a regular string", False),
+            ("Not a comment but a regular string", True),
```

Comment:
maybe add a case like "Name with spaces".
It is similar to the test Not a comment but a regular string, but it states more clearly what you want to test I think.
You could still keep the Not a comment but a regular string to also indicate a sentence will be seen as a name.

Reply:
Done